removeDuplicatesFromSortedList.py

```diff
@@ -0,0 +1,26 @@
+# Definition for singly-linked list.
+# class ListNode:
+#     def __init__(self, x):
+#         self.val = x
+#         self.next = None
+
+class Solution:
+    def deleteDuplicates(self, head):
+        """
+        :type head: ListNode
+        :rtype: ListNode
+        """
+        if not head:
```

Comment:
`if not head or not head.next`
When the list is empty or there is only one node, you can directly return `head`.